src/index.ts

```diff
@@ -2370,9 +2380,9 @@ function getLatestContractVersion({
 				return (partsB[1] || 0) - (partsA[1] || 0)
 			})
 
-		// If experimental is false, filter out versions that are not 'v4'
+		// If experimental is false, filter out versions don't match LATEST_STABLE_CONTRACT_VERSION exactly
 		if (!experimental && type === 'normal') {
-			versions = versions.filter((version) => version.startsWith(LATEST_STABLE_CONTRACT_VERSION))
+			versions = versions.filter((version) => version === LATEST_STABLE_CONTRACT_VERSION)
```

Comment:
We're making changes to functions that also have been changed in v0.4 branch, we'll have to fix some conflicts once we merge

Reply:
QQ